dubbo-common/src/main/java/org/apache/dubbo/common/utils/ReflectUtils.java

```diff
@@ -1273,16 +1272,18 @@ public static boolean isPublicInstanceField(Field field) {
         Class<?> beanClass = bean.getClass();
         BeanInfo beanInfo = null;
         T propertyValue = null;
+
+        beanClass.getDeclaredFields();
         try {
             beanInfo = Introspector.getBeanInfo(beanClass);
-            propertyValue = (T) Stream.of(beanInfo.getPropertyDescriptors())
-                    .filter(propertyDescriptor -> propertyName.equals(propertyDescriptor.getName()))
-                    .map(PropertyDescriptor::getReadMethod)
```

Comment:
May be use PropertyDescriptor::getReadMethod, but consider the getXXX and isXXX scenarios more better ?